lib/features/support/docker.rb

```diff
@@ -55,7 +65,7 @@ def compose_project_name
     def run_docker_compose_command(command, compose_file: COMPOSE_FILENAME, success_codes: nil)
       project_name = compose_project_name.nil? ? '' : "-p #{compose_project_name}"
       command = "docker-compose #{project_name} -f #{compose_file} #{command}"
-      Runner.run_command(command, success_codes: success_codes)
+      @last_exit_code = Runner.run_command(command, success_codes: success_codes)
```

Comment:
`Runner.run_command` returns an array of exit code & output. I've ended up using both for the source map work (for checking error messages), so this could be:

```suggestion
      @last_exit_code, @output = Runner.run_command(command, success_codes: success_codes)
```

with `output` added as an `attr_accessor` & a step for reading it, e.g. https://github.com/bugsnag/bugsnag-source-maps/blob/9ab060ab28d4dd3f13a6fbb9fdbfd6d3c6cdca2b/features/steps/steps.rb#L88-L97

Reply:
Ah, didn't realise you found both valuable, but that would be a good addition!